UnitTests/resources/ExpectedExceptionIntegrationTestCases.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using NUnit.Framework;
+using NUnit2To3SyntaxConverter.ExpectedException;
+
+namespace UnitTests
+{
+  public class ExpectedExceptionIntegrationTests
+  {
+    
+    [ExpectedException]
+    public void SimpleBaseCase ()
+    {
+      Console.WriteLine();
+    }
+
+    [ExpectedException (typeof (DivideByZeroException)]
+    public void WithCustomExceptionType ()
+    {
+      Console.WriteLine();
+    }
+    
+    [ExpectedException (typeof (ArgumentNullException)]

```

Comment:
How come he doesn't care about the missing closing parenthesis?

Reply:
because roslyn is too smart somtimes